docs/guides/dask-ray-task-runners.md

```diff
@@ -281,18 +279,18 @@ goodbye marvin
 The tasks are not submitted to the `DaskTaskRunner` and are run sequentially.
 
 
-## Running parallel tasks with Ray
+## Run parallel tasks with Ray
 
-To demonstrate the ability to flexibly apply the task runner appropriate for your workflow, use the same flow as above, with a few minor changes to use the [`RayTaskRunner`](https://prefecthq.github.io/prefect-ray/) where we previously configured `DaskTaskRunner`. 
+To flexibly apply the appropriate task runner for your workflow, use the same flow as above, with a few minor changes to use the [`RayTaskRunner`](https://prefecthq.github.io/prefect-ray/) where you previously configured `DaskTaskRunner`. 
 
-To configure your flow to use the `RayTaskRunner`:
+Configure your flow to use the `RayTaskRunner`:
 
 1. Make sure the `prefect-ray` collection is installed by running `pip install prefect-ray`.
 2. In your flow code, import `RayTaskRunner` from `prefect_ray.task_runners`.
 3. Assign it as the task runner when the flow is defined using the `task_runner=RayTaskRunner` argument.
 
 !!! warning "Ray environment limitations"
-    While we're excited about parallel task execution via Ray to Prefect, there are some inherent limitations with Ray you should be aware of:
+    While we're excited about parallel task execution through Ray to Prefect, there are some inherent limitations with Ray:
     
     - Support for Python 3.11 is [experimental](https://docs.ray.io/en/latest/ray-overview/installation.html#install-nightlies).
```

Comment:
- Ray currently officially supports x86_64, aarch64 (ARM) for Linux, and Apple silicon (M1) hardware.